HarryPotterApp/AppDelegate.swift

```diff
@@ -1,11 +1,17 @@
 import UIKit
+import Presentation
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        UINavigationBar.appearance().setBackgroundImage(UIImage(), for: .default)
+        UINavigationBar.appearance().shadowImage = UIImage()
+        UINavigationBar.appearance().tintColor = .white
+        //UINavigationBar.appearance().backgroundColor = Color.navigationBarBlack
```

Comment:
remove unused code.

Reply:
That did not have to be commented, I stayed like this because I was trying things, I accommodated it there.